test/gmpTransferAdapter/nativeTransfer/deposit.js

```diff
@@ -0,0 +1,185 @@
+// The Licensed Work is (c) 2022 Sygma
+// SPDX-License-Identifier: LGPL-3.0-only
+
+const TruffleAssert = require("truffle-assertions");
+const Ethers = require("ethers");
+const Helpers = require("../../helpers");
+
+const GmpTransferAdapterContract = artifacts.require("GmpTransferAdapter");
+const GmpHandlerContract = artifacts.require(
+  "GmpHandler"
+);
+const BasicFeeHandlerContract = artifacts.require("BasicFeeHandler");
+const FeeHandlerRouterContract = artifacts.require("FeeHandlerRouter");
+const XERC20FactoryContract = artifacts.require("XERC20Factory");
+const XERC20Contract = artifacts.require("XERC20");
+const XERC20LockboxContract = artifacts.require("XERC20Lockbox");
+
+
+contract("Gmp transfer adapter - [Deposit - native token]", async (accounts) => {
```

Comment:
GmpTransferAdapter does not support native token relays, so it seems this tests are invalid.

Reply:
The naming of tests is misleading, it transfers xERC20 but it's wrapped native token - updated the naming of the tests.